File: Iniciante/URI 1035.r
input = file('stdin', 'r')

line = strsplit(readLines(input, n=1), " ")
a = as.integer(line[[1]][1])
b = as.integer(line[[1]][2])
c = as.integer(line[[1]][3])
d = as.integer(line[[1]][4])

if (b > c && d > a && c > 0 && d > 0 && (c + d) > (a + b) && a %% 2 == 0) {
    write(sprintf("Valores aceitos"), '')
} else { 
    write(sprintf("Valores nao aceitos"), '')
}

File: Iniciante/URI 1035.php
<?php
    $line = explode(" ", trim(fgets(STDIN)));
    $A = intval($line[0]);
    $B = intval($line[1]);
    $C = intval($line[2]);
    $D = intval($line[3]);
    
    if (($B > $C) && ($D > $A) && (($C + $D) > ($A + $B)) && ($C > 0) && ($D > 0) && ($A % 2 == 0)) {
        echo "Valores aceitos\n"; 
    } else {
        echo "Valores nao aceitos\n";
    }
?>